modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -137,13 +137,18 @@ def submit_form_to_benefits_intake
           { form_number: params[:form_number], status:, uuid: confirmation_number }
         )
 
+        submission_archive_handler = SimpleFormsApi::S3Service::SubmissionArchiveHandler.new(
+          benefits_intake_uuid: confirmation_number
+        )
+        presigned_s3_url = submission_archive_handler.run
```

Comment:
I'm thinking `run` can return a `presigned_s3_url` and we can put that in the JSON response to the FE.